Autopilot/AttitudeManager/PID.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file PID.h
+ * @author Ian Frosst
+ * @date  March 26, 2017
+ * @copyright Waterloo Aerial Robotics Group 2017 \n
+ *   https://raw.githubusercontent.com/UWARG/PICpilot/master/LICENCE 
+ */
+
+#ifndef PID_H
+#define	PID_H
+
+#include "main.h"
+
+#define PID_RESET_TIME 1000 // timeout to reset I and D terms (ms)
```

Comment:
If you go too long without updating the PID, it resets the integral and the last derivative. Prevents integral windup and derivative spikes when on the ground.